src/DemoData.hs

```diff
@@ -156,6 +187,9 @@ universe rnd = void $ do
 
 -- * Helpers
 
+-- This 'Monad m => m a' is strange, why not simply 'a' instead?
```

Comment:
`Monad a` is used in favor of do notation and imperative style. No hard opinion here.